### ISSUE TITLE
Demo page:  "easeFrom" and "easeFromTo" throw errors

### ISSUE THREAD

**Issue by jeremyckahn** [joelambert/morf#3]
Click "easeFrom" and "easeFromTo" here : http://www.joelambert.co.uk/morf/

Check the JavaScript console, there are errors for "Invalid matrix string."  Not sure what's causing this.  My initial theory is that there is that the "e" values (meaning it's a really big number) in the CSS string are breaking the syntax, but I have not tested that.

Thoughts?

**Comment by joelambert** [comment]
Yeah this is caused by: [WebkitCSSMatrix.ext.js#L145](https://github.com/joelambert/morf/blob/master/js/src/WebkitCSSMatrix.ext.js#L145)

I found in testing (like you) that some of the strings generated were not valid - this was causing the animation to fail so I added the `try/catch` to make sure that the animation at least occurred, even though it was sub optimal & missing some keyframes.

I thought it is was to do with the 'e' in the number string as well - I think I tried rounding but this didn't work. Would be good to get to the bottom of this though!
